src/game/setGameUI.js

```diff
@@ -39,16 +39,19 @@ const updateGraphs = (history, hospitalCapacity) => {
     const fullYear = 365;
     const costHistory = [];
     const caseHistory = [];
-    history.forEach((entry) => {
-        const targetDate = new Date(Date.UTC(2020, 0, 1));
+
+    const visibleHistory = history.slice(13); // Hack to make thinks align well on screen
```

Comment:
Had to do a bit of a hack in order to get the data series to fit our alignment constraints. In this case we're shaving off a few days of the initial data so that everything aligns. Not great, but couldn't find a better alternative.